test/ng/directive/validatorsSpec.js

```diff
@@ -351,7 +351,7 @@ describe('validators', function() {
 
 
     it('should accept values of any length when maxlength is non-numeric', function() {
-      var inputElm = helper.compileInput('<input type="text" ng-model="value" ng-maxlength="{{maxlength}}" />');
+      var inputElm = helper.compileInput('<input type="text" ng-model="value" ng-maxlength="maxlength" />');
```

Comment:
I guess that this was a bad test in the first place is not really part of this PR? Perhaps, if so, then when you are squashing all this, we move this change into its own commit?